exercises/J-array-get-set/exercises2.js

```diff
@@ -6,14 +6,20 @@
   - change the first value in the array to the number 1
 */
 
+/*
 var numbers = [1, 2, 3]; // Don't change this array literal declaration
+numbers[3] = 4;
+*/
+
+var numbers = [1, 2, 3]; // Don't change this array literal declaration
+numbers[numbers.length] = 4;
```

Comment:
Hi Rafael, great work. Only suggestion I had was you could use the push function here:
numbers.push(4);